src/python/bot/fuzzers/afl/launcher.py

```diff
@@ -365,27 +369,6 @@ def decide_fast_cal_random(self, input_dir):
 
     assert None, 'This should not be reached'
 
-  def print_strategies(self):
-    """Print the strategies used for logging purposes."""
-    fuzzing_strategies = []
-
-    if self.generator_strategy == engine_common.Generator.RADAMSA:
-      fuzzing_strategies.append('strategy_' +
-                                strategy.CORPUS_MUTATION_RADAMSA_STRATEGY.name)
-    elif self.generator_strategy == engine_common.Generator.ML_RNN:
-      fuzzing_strategies.append('strategy_' +
-                                strategy.CORPUS_MUTATION_ML_RNN_STRATEGY.name)
-    if self.fast_cal == strategies.FastCal.RANDOM:
```

Comment:
This is no longer surfaced in the new stats pipeline (see line 312). However since stats were completely broken anyway this isn't necessarily a regression.

@jonathanmetzman is the assumption on line 312 correct that we can only get these when we fuzz? Are these stats important enough for us to make interface changes to support strategies obtained during fuzz() ?